test/daimon_skycrawlers/crawler_test.rb

```diff
@@ -3,14 +3,21 @@
 class DaimonSkycrawlersCrawlerTest < Test::Unit::TestCase
   sub_test_case 'fetch html' do
     setup do
-      @crawler = ::DaimonSkycrawlers::Crawler.new 'http://example.com'
+      stubs = Faraday::Adapter::Test::Stubs.new do |stub|
+        stub.get("/") {|env|
+          [200, {}, "body"]
+        }
+      end
+      @crawler = ::DaimonSkycrawlers::Crawler.new 'http://example.com' do |faraday|
+        faraday.adapter :test, stubs
+      end
     end
 
     def test_on_fetch
-      # TODO Mock it
-      @crawler.fetch 'http://example.com' do |url, response|
-        assert { url == 'http://example.com' }
-        assert response
+      @crawler.fetch "/" do |path, response|
+        assert_equal("/", path)
```

Comment:
Alternatively, `url` may be unnecessary as block argument.
Because `url` is accessible in this block.